manifests/overlays/prod/applications/thoth-station/prod/prod-slo-reporter-aws.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: prod-thoth-slo-reporter-aws
+spec:
+  project: thoth-stage
+  source:
+    repoURL: 'https://github.com/thoth-station/thoth-application.git'
+    path: slo-reporter/overlays/aws-prod-ocp4-stage
+    targetRevision: master
```

Comment:
For what it's worth I _think_ you don't need to include targetRevision when it is master

Reply:
i.e the default is master

Reply:
Sounds good, I can remove it. I wanted to leave it because I saw that [thoth-station will be moving to using main as the protected branch in most repositories](https://github.com/thoth-station/core/issues/391), and wanted it to be clear which is the default `targetRevision`. Any thoughts?

Reply:
I think we should leave it for now
let's change it all over all the app of thoth-station at once.